Jenkinsfile.integration

```diff
@@ -26,6 +26,7 @@ pipeline {
         SOCOK8S_DEVELOPER_MODE = "True"
         DEPLOYMENT_MECHANISM = "openstack"
         ANSIBLE_STDOUT_CALLBACK = "yaml"
+        USER = "jenkins" /* Why isn't this set in the jenkins environment? */
```

Comment:
are you sure that a trailing "/*" comment is actually going to work? as far as I remember the parsing of this is weird.

Reply:
I didn't know that jenkins and/or groovy have buggy comment parsing, is that documented somewhere?

Reply:
According to CI, this seems to work fine